source/includes/details-for-a-single-app.md

```diff
@@ -323,6 +325,7 @@ Field                    | Description
 **price**                 | The price of the app, as displayed on the store side.
 **price_currency**        | Price currency, according to ISO 4271 standard.
 **price_value**           | Price value, expressed as digits only.
+**privacy_url**           | Privacy policy url, shown on store for the publisher of this app, if present. ![android_only](../images/android_logo.jpg)
```

Comment:
Privacy policy URL provided by the publisher, if present.

Reply:
https://github.com/AppMonsta/slate/commit/629db906d323a1abae3c0b64568ed9443ed91e2d